packages/rpc-core/src/rpc-methods/getFirstAvailableBlock.ts

```diff
@@ -0,0 +1,10 @@
+import { RpcResponse, Slot } from './common';
+
+type GetFirstAvailableBlockApiResponse = RpcResponse<Slot>;
```

Comment:
This doesn't return an `RpcResponse` in fact. It's just the raw integer!

```ts
// Not an `RpcResponse<TValue>`...
{context: {...}, value: 1234}
// Actually just `TValue`
1234
```
```suggestion
type GetFirstAvailableBlockApiResponse = Slot;
```

Reply:
Well at least I got it right in my PR description :sweat: any idea about why this didn't get picked up by the compiler or the tests?

Reply:
Yeah! I mean we could write _Typescript_ tests, but that’s pretty next level.

Reply:
In case it wasn't clear what I meant by this, some people _actually_ have written tests that assert whether source code raises type errors or not. Pretty extreme.

Reply:
Haha don't worry, it was *very* clear. I didn't want to go there under any circumstances :sweat_smile: